pkg/virt-handler/vm.go

```diff
@@ -67,6 +69,60 @@ type VMHandlerDispatch struct {
 	host          string
 }
 
+func (c *VMHandlerDispatch) getPodIPAddress() (string, error) {
+	// NB the libvirtd image is currently still in the host
+	// network namespace, so we need to figure out the
+	// default local outbound IP address it would use. Here
+	// we assume virt-handler is also in the same network
+	// namespace as libvirtd, and use this trick of binding
+	// to a local UDP socket lets us figure out the IP addr.
+	// If libvirtd were to move into a private network
+	// namespace, then we can instead switch to query
+	// the Status.PodIP field from the libvirtd POD.
```

Comment:
I'd like to perform the switch to pod networking rather sooner than later.

This will directly break network connectivity, as we'll only have "ipd endpoints" inside the pod, and not a complete interface/L2 connectivity.

We might be able to workaround this by using macvlan/bridge or alike.